root/static/js/canto-modules.js

```diff
@@ -1383,7 +1383,7 @@ var pubmedIdStart =
           searchId: null,
           results: null,
         };
-        $scope.userIsAdmin = CantoGlobals.current_user_is_admin;
+        $scope.userIsAdmin = CantoGlobals.is_admin_user == "1";
```

Comment:
At some point we should change all of the CantoGlobals var to boolean.  :-)

Reply:
There's an issue open for this at #2209. Seems `is_admin_user` is the only property that has this problem at the moment.